vundo.el

```diff
@@ -365,10 +374,8 @@ If MOD-LIST non-nil, extend on MOD-LIST."
             (cl-incf uidx))
           ;; If this modification contains a timestamp, the previous
           ;; state is saved to file.
-          (when (and mod-timestamp (not pos-only))
-            (setf (vundo-m-prev-saved-ts (car new-mlist)) mod-timestamp)
-            (when (time-less-p vundo--latest-saved-ts mod-timestamp)
-	      (setq vundo--latest-saved-ts mod-timestamp))))))
+          (when (and mod-timestamp (not pos-only) vundo-highlight-saved-nodes)
```

Comment:
I don't think we should check for `vundo-highlight-saved-nodes` here. Checking this doesn't bring any advantage. If we don't check it, a) the `prev-node-saved-ts` would be always correct, and b) if there user changes vundo-highlight-saved-nodes, we don't need to regenerate the mod list. Also the field can be changed back to a boolean now, right?

Reply:
Sounds good. 

> the field can be changed back to a boolean 

I store the TS despite using it as a boolean so we can report on the change time (see changes below to `i`).  I would personally enjoy seeing a little message like "Saved 32min ago" when I move over a saved node.